src/qmapshack/gis/qms/serialization.cpp

```diff
@@ -718,6 +731,7 @@ QDataStream& CGisItemWpt::operator<<(QDataStream& stream)
     setIcon();
     setText   (CGisListWks::eColumnName, getName());
     setToolTip(CGisListWks::eColumnName, getInfo(IGisItem::eFeatureShowName));
+    IGisItem::updateDecoration(eMarkNone, eMarkNone);
```

Comment:
Why?

Reply:
It causes the symbols for the tags to appear. I should move it into the `if(version){}`.
Or do you want it to be somewhere else?

Reply:
Then we have duplicated code. Probably all set... methods are covered by updateDecoration, too.